From ed15561eb0ca3a65267612ad62e4ea9eecc720ae Mon Sep 17 00:00:00 2001
From: Ran Shalit <rshalit@habana.ai>
Date: Wed, 09 Dec 2020 11:01:06 +0200
Subject: [PATCH] [SW-0] gaudi: fix clock gating initialization

Clock gating should be initialized only when security
is disabled, so the logic is fixed.

Change-Id: Ie753aa638a05325a60b4cdaca4713a7b7810a8b9
Signed-off-by: Ran Shalit <rshalit@habana.ai>
(cherry picked from commit 11c28f0a0bf6f001fd86ac4f47c7d37d8fcd32fc)
---

diff --git a/preboot/preboot.c b/preboot/preboot.c
index 13904e6..0853cef 100644
--- a/preboot/preboot.c
+++ b/preboot/preboot.c
@@ -282,7 +282,7 @@
 
 	preboot_led_on();
 	is_sec_en = preboot_security_is_enabled();
-	if (is_sec_en)
+	if (!is_sec_en)
 		hl_clock_gating_init();
 
 	sram_scrambling_init();
